flink-connector-kudu/src/main/java/org/apache/flink/streaming/connectors/kudu/connector/KuduConnector.java

```diff
@@ -133,8 +136,15 @@ public void close() throws Exception {
 
         if (client == null) return;
         client.close();
+        
+        if (session == null) return;
```

Comment:
you should close session first and then client no?

Reply:
thanks for your review, have been fixed